modules/collections/client/services/collections.client.services.js

```diff
@@ -12,11 +12,11 @@ angular.module('collections').factory('CollectionModel', function (ModelBase) {
       return this.put('/api/collections/' + collectionId + '/remove', {});
     },
 
-    addDocument: function(collectionId, documentId, docType) {
+    addDocument: function(collectionId, documentId, sortOrder, docType) {
       if ( docType === "main") {
-        return this.put('/api/collections/' + collectionId + '/document/' + documentId + '/main/add', {});
+        return this.put('/api/collections/' + collectionId + '/document/' + documentId + '/main/add', { sortOrder : sortOrder});
```

Comment:
Added the sortOrder to the request body, so that the parameter is optional and the APIs will not require updating.